hivelvet-backend/app/config/routes.ini

```diff
@@ -77,5 +77,6 @@ DELETE  @label_delete   : /labels/@id            = Actions\Labels\Delete->execut
 ; rooms routes
 GET     @rooms_index    : /rooms/@user_id        = Actions\Rooms\Index->show
 POST    @rooms_add      : /rooms                 = Actions\Rooms\Add->save
-PUT     @rooms_edit     : /rooms/@id             =  Actions\Rooms\Edit->rename
-DELETE  @room_delete    : /rooms/@id             = Actions\Rooms\Delete->execute
\ No newline at end of file
+PUT     @rooms_edit     : /rooms/@id             = Actions\Rooms\Edit->rename
+GET     @rooms_start    : /rooms/start/@id       = Actions\Rooms\Start->execute
```

Comment:
Can we use POST instead of GET for room start? It will prevent most of common users from starting it by mistake using a browser.